ts/features/walletV3/payment/screens/WalletPaymentOutcomeScreen.tsx

```diff
@@ -1,28 +1,43 @@
-import { useNavigation } from "@react-navigation/native";
+import { RouteProp, useNavigation, useRoute } from "@react-navigation/native";
 import React from "react";
 import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
 import {
   AppParamsList,
   IOStackNavigationProp
 } from "../../../../navigation/params/AppParamsList";
+import { WalletPaymentParamsList } from "../navigation/params";
+import { WalletPaymentOutcome } from "../types/PaymentOutcomeEnum";
+
+type WalletPaymentOutcomeScreenNavigationParams = {
+  outcome: WalletPaymentOutcome;
+};
+
+type WalletPaymentOutcomeRouteProps = RouteProp<
+  WalletPaymentParamsList,
+  "WALLET_PAYMENT_OUTCOME"
+>;
 
 const WalletPaymentOutcomeScreen = () => {
+  const { params } = useRoute<WalletPaymentOutcomeRouteProps>();
   const navigation = useNavigation<IOStackNavigationProp<AppParamsList>>();
 
   const handleContinue = () => {
     navigation.popToTop();
     navigation.pop();
   };
+
   return (
     <OperationResultScreenContent
       title="Hai pagato xx,xx €"
+      subtitle={params.outcome.toString()}
       action={{
-        label: "Ok chiudi",
+        label: "Ok, chiudi",
```

Comment:
Why not use a locale string? what do you think to add an item to locale files and use i18n?

Reply:
This screen is not ready yet. We are waiting for the final design with the errors mapping to implement the correct strings and pictograms :)